portfolio/src/main/webapp/script.js

```diff
@@ -76,14 +76,21 @@ function getComments() {
  * extracts numComment from URL and then displays comment
```

Comment:
This comment should be updated to reflect the new functionality in initComments()